### PR TITLE
Fix static padding calculation

### PR DESCRIPTION
**Pull request by AlexDut**
Hello,

I was testing the models and found that the results were different between the EfficientNet model with static padding convolutions and the EfficientNet model with dynamic padding convolutions.
When I looked into the code I noticed that there was a difference in the padding's calculation.

The appropriate calculation is the one used for the dynamic padding ie:
`[pad_w // 2, pad_w - pad_w // 2, pad_h // 2, pad_h - pad_h // 2]` ([code](https://github.com/lukemelas/EfficientNet-PyTorch/blob/master/efficientnet_pytorch/utils.py#L240))

You can test it with the quick & dirty script I made:
```Python
import torch
import torchvision.transforms as transforms
from efficientnet_pytorch import EfficientNet

# Tensorflow-like EfficientNet pre-process step
test_input = torch.ones((1, 224, 224, 3))
test_input = test_input/255
mean = torch.Tensor([0.485, 0.456, 0.406])
std = torch.Tensor([0.229, 0.224, 0.225])
test_input = (test_input - mean) / torch.sqrt(std)
test_input = test_input.permute(0,3,2,1)

model = EfficientNet.from_pretrained('efficientnet-b0', image_size=None)  # Model using Conv2dDynamicSamePadding
model.eval()

model_wrong = EfficientNet.from_pretrained('efficientnet-b0') # Model using Conv2dStaticSamePadding
model_wrong.eval()

print(model(test_input).sum())
print(model_wrong(test_input).sum())
```

### CODE REVIEW

**Comment by lukemelas** [comment]
Thank you so much for spotting this! Merging.
